composer.json

```diff
@@ -138,7 +138,7 @@
         "spryker/oauth-api": "^1.3.0",
         "spryker/oauth-auth0": "^1.1.0",
         "spryker/oauth-backend-api": "^1.4.0",
-        "spryker/oauth-client": "^1.3.0",
+        "spryker/oauth-client": "^1.4.0",
```

Comment:
I don't see spryker/oauth-client requirement on the PBC list of required modules (is there a reason for the upgrade? or is the list on PBC manifest outdated?).

Reply:
`spryker/tax-app` requires `^1.4.0`.

```
mads@xps:~/projects/demoshops/b2c-demo-shop$ composer why spryker/oauth-client
spryker-shop/b2c-demo-shop dev-master requires spryker/oauth-client (^1.4.0) 
spryker/app-catalog-gui    1.3.0      requires spryker/oauth-client (^1.0.0) 
spryker/tax-app            0.1.0      requires spryker/oauth-client (^1.4.0)
```